Northstar.CustomServers/mod/scripts/vscripts/mp/_gamestate_mp.nut

```diff
@@ -184,11 +184,17 @@ void function GameStateEnter_PickLoadout_Threaded()
 	// titan selection menu can change minPickLoadOutTime so we need to wait manually until we hit the time
 	while ( Time() < GetServerVar( "minPickLoadOutTime" ) )
 		WaitFrame()
-	
+
+	//added by Khalmee for the sake of fixing LTS spectator mode
+	foreach ( player in GetPlayerArray() ){
+		if ( IsPrivateMatchSpectator( player ) )
+		InitialisePrivateMatchSpectatorPlayer( player )
+	}
```

Comment:
```suggestion
	foreach ( player in GetPlayerArray() )
	{
		if ( IsPrivateMatchSpectator( player ) )
			InitialisePrivateMatchSpectatorPlayer( player )
	}
```

Reply:
Addressed in c794c84753d94b0484d6772cab38ee71c0c2d881